python/tk_multi_publish2/progress/progress_handler.py

```diff
@@ -180,7 +180,9 @@ def process_log_message(self, message, status, action):
 
         item.setText(0, message)
         item.setToolTip(0, message)
-        item.setIcon(0, icon)
+
+        if icon:
+            item.setIcon(0, icon)
```

Comment:
same comment as for the maya PR - i think we either need a subfolder for the icons used by the hooks or perhaps even store them in the hook folder.

Reply:
Agree. Addressed similarly to maya engine.